README.md

```diff
@@ -287,3 +281,7 @@ To classify documents, you must [create a custom document classifier in the Goog
 - You can use the [test documents and forms](sample-deployments/composer-orchestrated-process/documents-for-testing/forms-to-train-docai) to train and evaluate the classifier in your GCP environment.
 
 - We have created an annotated dataset to expedite the training process. Please contact your Google account representative to get access to the annotated dataset.
+
+- The output labels of the classifier **MUST** match the configured labels in the composer DAG configuration `doc-ai-processors`. Out of the box, the solution supports `form` and `invoice` labels. Any other label would cause the flow to treat the document as a generic document and will process it without extracting structured data from the document.
+
+- If you configure the classifier identifier (in the `projects/<CLASSIFIER_PROJECT>/locations/<CLASSIFIER_LOCATION>/processors/<CLASSIFIER_ID>` form) in the terraform variable `customer_classifier`, terraform will push the classifier ID to composer as a default argument.
```

Comment:
Nit: small rewrite on the last bullet point for prescriptive guidance.

"""
After training the custom classifier, set the classifier ID to composer as a default argument. Add the following variable to your Terraform variables file and run `terraform apply` again.

```sh
classifier_project= # projects/<CLASSIFIER_PROJECT>/locations/<CLASSIFIER_LOCATION>/processors/<CLASSIFIER_ID>
```
"""

Reply:
makes sense - thanks